eclass/bitcoincore.eclass

```diff
@@ -12,7 +12,7 @@
 #
 # The eclass provides all common dependencies as well as common use flags.
 
-has "${EAPI:-0}" 5 || die "EAPI=${EAPI} not supported"
+has "${EAPI:-0}" 5 6 || die "EAPI=${EAPI} not supported"
```

Comment:
From a quick look, this eclass is not EAPI 6-ready. It's still full of deprecated `epatch` calls.